src/components/Main.tsx

```diff
@@ -265,27 +266,27 @@ export const Main = () => {
   useAsyncEffect(
     async () => {
       if (isMaestro()) return
-      setLegacyLanding((await getExperimentConfigValue('landingType')) === 'A_legacy')
+      setExperimentConfig(await getExperimentConfig())
```

Comment:
This data should be supplied by a Provider (context API) instead of passed around as a route param. I propose you move this expirement config initialization out of Main and into its own Provider. Then use the context in any scene where its needed.